.gitignore

```diff
@@ -9,6 +9,7 @@
 
 # dependencies
 /node_modules
+/**/node_modules
 
```

Comment:
you also might want to add `.angular`, which is a cache on Angular 13.

Reply:
Added.